.github/workflows/test.yaml

```
@@ -10,9 +10,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
-        django-version: ['2.2', '3.1', '3.2']
-        postgres-version: ['9', '10', '11', '12', '13']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        django-version: ['2.2', '3.2', '4.0']
+        postgres-version: ['10', '11', '12', '13']
```

Django 4 requires Python>=3.8:

```suggestion
        postgres-version: ['10', '11', '12', '13']
        exclude:
          - python-version: 3.7
            django-version: 4.0 
```